examples/vue/src/pages/ui/components/authenticator/sign-in-with-phone/index.vue

```diff
@@ -3,9 +3,23 @@ import { Amplify } from 'aws-amplify';
 import { I18n } from 'aws-amplify/utils';
 import { Authenticator, translations } from '@aws-amplify/ui-vue';
 import '@aws-amplify/ui-vue/styles.css';
-import aws_exports from './aws-exports';
 
-Amplify.configure(aws_exports);
+const amplifyOutputs =
+  import.meta.env.VITE_VERSION === 'gen1'
+    ? (
+        await import(
+          // @ts-ignore
```

Comment:
We should just have a util that resolves the import prefix (e.g. `@environments/auth/auth-with-phone-number/` and appends the output file type. This pattern of repetition is fragile

Reply:
Yeah, originally I was doing something to simplify it but Vite has some trouble doing import analysis when it comes to variables in dynamic imports due to the limitations of the [plugin](https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations) it is using. e.g.:
- Variables only represent file names one level deep (if file is `src/aws-exports.js`, the import would fail)
- Imports must end with a file extension
- Imports must start with `./` or `../`
etc...

But I like the idea to create a util. Let me try to see if I can create a util and simplify this as much as possible without annoying Vite.

Reply:
Got it, thanks for confriming